week4_ts/src/App.tsx

```diff
@@ -0,0 +1,15 @@
+import Router from "./core/router";
+import { ThemeProvider } from "styled-components";
+import GlobalStyle from "./styles/globalStyle";
+import theme from "./styles/theme";
+
+export default function App() {
+    return (
+        <>
+            <GlobalStyle />
+            <ThemeProvider theme={theme}>
+                <Router />
+            </ThemeProvider>
```

Comment:
```suggestion
            <ThemeProvider theme={theme}>
                <GlobalStyle />
                <Router />
            </ThemeProvider>
```

`GlobalStyle` 도 안에 위치시키면 theme 사용이 가능해집니다!